src/main/resources/jenkins/branch/MultiBranchProject/orphanedItemStrategyDescription.properties

```diff
@@ -0,0 +1,3 @@
+blurb=Jobs for removed heads can be removed immediately or kept based on a desired retention strategy. \
+   By default, jobs will be removed as soon as the folder computation determines they are no longer present.\
+   But it may be useful to keep them in order to examine build results of a branch after it&apos;s been removed, for example.
```

Comment:
```suggestion
   As an example, it may be useful to configure a different retention strategy to be able to examine build results of a branch after it has been removed.
```

Reply:
Looks like GitHub got confused and thinks this is outdated, but it isn't?

Reply:
Maybe I merged stuff in the wrong order?

Reply:
Added it manually